client/src/components/GenreButton.js

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+const GenreButton = styled.button`
+  width: 8.3rem;
+  height: 2.75rem;
+  border: none;
+  color: #000;
+  font-family: 'Quicksand Book', sans-serif;
+  font-size: 0.9rem;
+  text-transform: uppercase;
+  background: transparent;
+  border-radius: 1rem;
+  padding-top: 5px;
+  margin-right: 10px;
+  margin-top: 10px;
```

Comment:
I don't think you need these since you got the grid to take care of positioning, in fact `margin-right` sets the buttons off-center with the "match me" button

Reply:
you are right! I first used flex-box on the wrapper element, so I forgot to delete these lines.